packages/react-form/src/useForm.tsx

```diff
@@ -75,7 +75,10 @@ export function useForm<TData>(opts?: FormOptions<TData>): FormApi<TData> {
   })
 
   formApi.useStore((state) => state.isSubmitting)
-  formApi.update(opts)
+
+  useEffect(() => {
+    formApi.update(opts)
+  }, [])
```

Comment:
We probably want to have `opts` in the dep array here

Reply:
I saw the wouldn't be needed because this method is called one time. This way it's called when the app is mounted.

If `opts` be a dep will be called always the forms submitted. This `update` method is only for `defaultValues`.